src/Bundle/test/src/Grid/AuthorWithBooksWithUseOutputWalkersDisabled.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Grid;
+
+use App\Entity\Author;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+final class AuthorWithBooksWithUseOutputWalkersDisabled extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public static function getName(): string
+    {
+        return 'app_author_with_books_with_use_output_walkers_disabled';
+    }
+
+    public static function getResourceClass(): string
+    {
+        return Author::class;
```

Comment:
This is another place which will block us from usage of this feature in Sylius itself :( But perhaps we could introduce registry, which will use non static function and use class name as parameters through injection